core/pkg/core/storage/schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE detection_status (
+    request_id VARCHAR(255) PRIMARY KEY,
+    status VARCHAR(255) NOT NULL,
```

Comment:
Need to decide what to do with status. I would prefer it to be enum with mapping to int, not varchar

Reply:
Let's try to do this way 

https://github.com/volatiletech/sqlboiler?tab=readme-ov-file#enums